### PR TITLE
Overlay transfer

### PR DESCRIPTION
**Pull request by BenG-SFDC**

### CODE REVIEW

File: ffsite/static/ffsite/overlays/scrolltext.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+			console.log('invalid value for numDonations, defaulting to 3');
+			numDonations = 3;
+		}
+
+		/* System variables, don't edit */
+		var currentAPIRes;
+		var initialAPIRes;
+		var updateCount = 0;
+		// Donation source, url of donations to pull
+		var jsonURL =  'https://' + window.location.host + '/d/donations?orderBy=';
```
Comment (AevumDecessus) [inline ffsite/static/ffsite/overlays/scrolltext.html]: Shouldn't this be ```window.location.hostname```? The [W3C spec](https://www.w3schools.com/js/js_window_location.asp) doesn't list ```window.location.host``` as an option

Comment (AevumDecessus) [inline ffsite/static/ffsite/overlays/scrolltext.html]: Shouldn't this be ```window.location.hostname```? The [W3C spec](https://www.w3schools.com/js/js_window_location.asp) doesn't list host, just hostname

Comment (BenG-SFDC) [reply ffsite/static/ffsite/overlays/scrolltext.html]: its actually origin, to ensure proper protocol is used as well

Comment (AevumDecessus) [reply ffsite/static/ffsite/overlays/scrolltext.html]: Actually it's ```location.origin``` not ```window.location.origin```
[Spec Docs](https://www.w3schools.com/jsref/prop_loc_origin.asp)

**Pull request merged**